packages/react/src/components/Storage/FileUploader/Tracker/index.tsx

```diff
@@ -54,106 +54,86 @@ export function Tracker({
-              flex="1"
-              gap="0"
-            >
-              <Flex
-                direction="row"
-                className="amplify-fileuploader__file-content"
-                gap="xxs"
-              >
-                <Text
-                  as="span"
-                  fontWeight="bold"
-                  className="amplify-fileuploader__filename"
-                >
-                  {name}
-                </Text>
+          )}
 
-                {showonStartEdit() && (
-                  <Button onClick={onStartEdit} size="small" variation="link">
-                    <EditIcon fontSize="medium" />
-                  </Button>
-                )}
-                <Text as="span" color="font.tertiary" marginInlineStart="small">
-                  {humanFileSize(size, true)}
-                </Text>
-              </Flex>
-              <FileState
-                error={isError}
-                errorMessage={errorMessage}
-                success={isSuccess && !isError}
-                paused={isPaused}
-                loading={isLoading}
-              />
-            </Flex>
-            {isLoading && (
-              <>
-                {isPaused ? (
-                  <Button onClick={onResume} size="small" variation="link">
-                    {translate('Resume')}
-                  </Button>
-                ) : (
-                  <Button onClick={onPause} size="small" variation="link">
-                    {translate('pause')}
-                  </Button>
-                )}
-              </>
-            )}
-            {/* {isSuccess && !isError && (
+          <Text as="span" className={ComponentClassNames.FileUploaderFileSize}>
+            {humanFileSize(size, true)}
+          </Text>
+          {isLoading && (
+            <>
+              {isPaused ? (
+                <Button onClick={onResume} size="small" variation="link">
+                  {translate('Resume')}
+                </Button>
+              ) : (
+                <Button onClick={onPause} size="small" variation="link">
+                  {translate('pause')}
+                </Button>
+              )}
+            </>
+          )}
+          {/* {isSuccess && !isError && (
               <Button size="small" onClick={onDelete}>
                 Delete
               </Button>
             )} */}
-            {/* {!isSuccess && !isLoading && ( */}
-            {!isLoading && (
-              <Button size="small" onClick={onCancel}>
-                <Text>
-                  <CloseIcon />
-                </Text>
-              </Button>
-            )}
-          </>
-        )}
-      </Flex>
-      <Flex direction="column" gap="0" alignItems="flex-end">
+          {/* {!isSuccess && !isLoading && ( */}
+          {!isLoading && (
+            <Button size="small" onClick={onCancel}>
+              <IconClose />
+            </Button>
+          )}
+        </>
+      )}
+      {isLoading ? (
```

Comment:
nit: If we use `{xxx && xxx}` in the same file above, should we keep the pattern consistent?